sdk/tables/data-tables/review/data-tables.api.md

```diff
@@ -480,6 +489,22 @@ export interface TableResponseProperties {
     tableName?: string;
 }
 
+// @public (undocumented)
+export interface TablesBatch {
+    // (undocumented)
+    createEntities: <T extends object>(entitites: TableEntity<T>[]) => Promise<void>;
```

Comment:
One possible solution here might be to have the batch itself just maintain a list of operations internally, so that all these CRUD methods just push the operations onto that.  Then, when submit batch is called, we "replay" these operations against the internal client to build the batch and then submit it.

Or perhaps, have these methods be void returning and just collect the promises from the internal call in an array and then the first thing SubmitBatch does is await them all in order?

Reply:
Thank you @ellismg because I had this same thought early on but got distracted and forgot to leave a comment about it. It feels strange that they are async to me as well.